packages/medusa/src/api-v2/admin/inventory-items/[id]/location-levels/route.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  ContainerRegistrationKeys,
+  remoteQueryObjectFromString,
+} from "@medusajs/utils"
+import { MedusaRequest, MedusaResponse } from "../../../../../types/routing"
+
+import { AdminPostInventoryItemsItemLocationLevelsReq } from "../../validators"
+import { MedusaError } from "@medusajs/utils"
+import { createInventoryLevelsWorkflow } from "@medusajs/core-flows"
+import { defaultAdminInventoryItemFields } from "../../query-config"
+
+export const POST = async (
+  req: MedusaRequest<AdminPostInventoryItemsItemLocationLevelsReq>,
+  res: MedusaResponse
+) => {
+  const { id } = req.params
+
+  const remoteQuery = req.scope.resolve(ContainerRegistrationKeys.REMOTE_QUERY)
+
+  const locationId = req.validatedBody.location_id
+
+  const locationQuery = remoteQueryObjectFromString({
+    entryPoint: "stock_location",
+    variables: {
+      id: locationId,
+    },
+    fields: ["id"],
+  })
+
+  const [stock_location] = await remoteQuery(locationQuery)
+
+  if (!stock_location) {
+    throw new MedusaError(
+      MedusaError.Types.NOT_FOUND,
+      `Stock location with id: ${locationId} not found`
+    )
+  }
```

Comment:
sugg: This validations feels like it should be in the workflows since they're loosely tying 2 modules together